resources/js/city.js

```diff
@@ -0,0 +1,64 @@
+import $ from "jquery";
+
+const base_url = import.meta.env.VITE_BASE_URL;
+
+const modal = document.getElementById("modal");
+const closeModalButton = document.getElementById("closeModalButton");
+
+closeModalButton.addEventListener(
+    "click",
+    () => (modal.style.display = "none")
+);
+
+document.addEventListener("DOMContentLoaded", function () {
+    const openModalButtons = document.querySelectorAll("#openModalButton");
+
+    openModalButtons.forEach((button) => {
+        button.addEventListener("click", function () {
+            const airlineId = button.getAttribute("data-id");
+
+            modal.style.display = "block";
+            document.getElementById("hidden").value = airlineId;
+        });
+    });
+});
+
+document.addEventListener("DOMContentLoaded", function () {
+    document
+        .getElementById("updateForm")
+        .addEventListener("submit", function () {
+            const cityId = document.getElementById("hidden").value;
+            const name = document.getElementById("updateName").value;
+
+            const formData = {
+                name: name,
+            };
+
+            $.ajax({
+                url: `${base_url}/api/cities/${cityId}`,
+                type: "PUT",
+                data: formData,
+            });
```

Comment:
```suggestion
        .addEventListener("submit", function (e) {
            const cityId = document.getElementById("hidden").value;

            $.ajax({
                url: `${base_url}/api/cities/${cityId}`,
                type: "PUT",
                data: $(this).serialize(),
                contentType: 'application/json',
                dataType: 'json'
            });
```
You can do this instead (serialize a form). This prevents you from having to manually map the data into an object.

Reply:
I think It's no possible in this case (I proved it)